R/json_nest.R

```diff
@@ -24,11 +24,120 @@ json_nest <- function(.data, ..., .names_sep = NULL) {
 json_nest.data.frame <- function(.data, ..., .names_sep = NULL) {
   check_suggested("jsonlite", use = TRUE, top_level_fun = "json_nest")
   dot_nms <- ...names()
-  # `{tidyr}` only warns but since we don't need backward compatibility we're
-  #   better off failing
-  if (is_null(dot_nms) || "" %in% dot_nms) {
-    abort("All elements of `...` must be named.")
-  }
```

Comment:
Why is this gone now?

Reply:
It happened in 2f9f557c108c7d3c5c3119ed877736937ebc712f when refactoring, it might have been a mistake, I'll put it back